src/scalib/preprocessing/quantization.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+
+
+class QUANTIZER:
+    r"""Quantize a side channel traces given as an array of float into an array of int16.
+        The quantizer estimates a shift and scale that minimize the loss due to the rounding operation.
+
+    .. math::
+        \mathrm{Quantize}( x) = (x - \mathrm{Shift}) \cdot \mathrm{Scale}
+
+    The shift and scale are computed using `n` samples as
+
+    .. math::
+        \mathrm{Shift} = \frac{1}{2} (\max_{i=1}^n x_i + \min_{i=1}^n x_i) \qquad and \qquad  \mathrm{Scale} = \frac{2^{14}}{\max_{i=1}^n x_i - \min_{i=1}^n x_i}.
+
+
+    Parameters
+    ----------
+    ns : int
```

Comment:
`ns` isn't needed: it can be inferred from the other parameters

Reply:
Indeed,  I removed it.